client/src/components/Categories/CategoriesOverview/CategoriesOverview.js

```diff
@@ -11,14 +11,26 @@ class CategoriesOverview extends Component {
         }
     }
     componentDidMount() {
-        apiClient.getCategories()
+        let language=localStorage.getItem("language");
+         apiClient.getCategories()
             .then(({ data }) => {
-                this.setState({
+                 data.map((category)=>{
+                    if(language==="Ar"){
```

Comment:
you should have language be small letters, so all of that `if` `else` can become:
`category.title = category.titleTranslation[language]

Reply:
@kabaros we do not have category title translation English

Reply:
ok .. have an `if` for english and the rest can be a lookup like I mentioned .. so that we're not restricted to two languages only

Reply:
updated

Reply:
@kabaros